packages/cli/generators/relation/belongs-to-relation.generator.js

```diff
@@ -122,7 +154,7 @@ module.exports = class BelongsToRelationGenerator extends BaseRelationGenerator
   }
 
   _getRepositoryRelationPropertyName() {
-    return utils.camelCase(this.artifactInfo.dstModelClass);
+    return this.artifactInfo.relationName;
```

Comment:
I guess this line fixes https://github.com/strongloop/loopback-next/issues/4209?

Reply:
See comment https://github.com/strongloop/loopback-next/pull/4373/files#r365448472. This line is part of fix.